### ISSUE TITLE
Create a basic bank transfer payment module

### ISSUE THREAD

**Issue by indico-bot** [indico/indico#1078]
``` yaml
# Ticket imported from Trac
# Reported by: pedersen
```

To allow bank transfer as a payment option, a basic bank transfer module has been created. It will generate a page and email with the necessary account information.

Concerning the attached patches:
- is the file htdocs/bankTransfer.py really necessary? I modeled it after the  files for paypal and co.
- the second patch adds a hook to inject text from the plugin to paymentDetails.

**Comment by ThiefMaster** [comment]
"Basic bank transfer" is covered by the `payment_manual` plugin. No dynamic fields to be inserted in the message though.

**Comment by pferreir** [comment]
Yeah, but we agreed that in the future we'd allow better customisation of this plugin, and the tags can come in handy. @OmeGak, did you create a ticket for this, by any chance?

**Comment by ThiefMaster** [comment]
Nope, I don't think there's a ticket for it yet.

**Comment by OmeGak** [comment]
I didn't create any ticket. Maybe I can do so before we completely forget about the ideas we discussed before Christmas and link to this issue.

**Comment by pferreir** [comment]
Yes, please! And then we will follow this up over there.

**Comment by OmeGak** [comment]
I still think that a standalone bank transfer method would be something to be discussed separately, since it could be greatly overhauled with extra logic.

The way I imagine it, inspired by the example of Transferwise:
![](http://files.bertaland.webnode.com/200000181-3e6373f5ad/tw6.png)

When the registrant clicks on "I've sent the money*, the manager could be notified by email or it could be displayed in some event management page. This could be useful for managers to know when to check the bank account for incoming payments.
